File: linguagens/Python/Exercicios/idade.py
idade1: int 
idade2: int 
media: float

nome1: str
nome2: str
n: int 
somai: int 
somai = 0

n = int(input("Quantas pessoas vao ser: "))

veti: [int] = [0 for x in range(n)] #vetor para idade 
vetn: [str] = [0 for y in range(n)] # vetor para o nome



for i in range (0,n):

    vetn = input(f"Nome da pessoa {i + 1}: ")
    veti = int(input(f"Idade da  pessoa {i + 1}: "))

    somai = somai + veti

media = (somai)/n

print(f"A media das Pessoas é: {media}")

File: linguagens/C++/exercicios/idade.cpp
#include <iostream> 
#include <string> 
#include <climits> 
using namespace std; 

int main()
{
    string nome1, nome2; 
    int idade1, idade2;
    float media;  

    cout << "Primeira pessoa: "; 
    getline(cin, nome1);

    cout << "Idade da primeira pessoa: "; 
    cin >> idade1;
    
    cin.ignore(INT_MAX, '\n'); // clear the buffer

    cout << "Segunda pessoa: "; 
    getline(cin, nome2);
    
    cout << "Idade da Segunda pessoa: "; 
    cin >> idade2;

    media = (idade1 + idade2) / 2;
    
    cout << "A idade de " << nome1 << " eh " << nome2 << " e de " << media << " anos" << endl;  


    return 0; 
}